packages/desktop-client/src/hooks/useSelected.tsx

```diff
@@ -21,9 +27,35 @@ function iterateRange(range, func) {
   }
 }
 
-export default function useSelected(name, items, initialSelectedIds) {
+type State = {
+  selectedRange: Range<string> | null;
```

Comment:
I would've expected, no?

```suggestion
  selectedRange: Range<number> | null;
```

Reply:
The reducer sets `selectedRange` to be `id`s. Which are actually `string`s. So technically the current implementation is correct. But it was a surprise for me too. :)